test/graphql/Offer/queries/getOffers.test.ts

```diff
@@ -83,11 +98,45 @@ describe("getOffers", () => {
     ]));
 
     it("returns no offers when no offers were created", async () => {
-      const { apolloClient } = await testClientFactory.user();
+      const { apolloClient } = await testClientFactory.applicant({
+        status: {
+          approvalStatus: ApprovalStatus.approved,
+          admin: await admins.next().value
+        }
+      });
       const { data, errors } = await apolloClient.query({ query: GET_OFFERS });
 
       expect(errors).toBeUndefined();
       expect(data!.getOffers).toHaveLength(0);
     });
   });
+
+  it("throws an error when the user is from a company", async () => {
```

Comment:
Lo escribiria asi:
`returns an error when the user is from a company`

ya que graphql retorna una response con los errors.